src/ArmTemplates/Extractor/EntityExtractors/Abstractions/EntityExtractorBase.cs

```diff
@@ -18,7 +24,7 @@ public abstract class EntityExtractorBase
         static readonly IMemoryCache Cache = new MemoryCache(new MemoryCacheOptions());
```

Comment:
Just like elsewhere. Can these be properties with private/no setters instead?

Reply:
Fixed this in ApiClientBase. Once all extractors will migrate to this class, will delete `EntityExtractorBase` at all